packages/fluentui/react-northstar/src/themes/teams/components/Chat/chatMessageStylesComfyRefresh.ts

```diff
@@ -0,0 +1,232 @@
+import { isNil } from 'lodash';
+import { ComponentSlotStylesPrepared, ICSSInJSStyle } from '@fluentui/styles';
+import { getBorderFocusStyles } from '../../getBorderFocusStyles';
+import { chatMessageSlotClassNames, ChatMessageStylesProps } from '../../../../components/Chat/ChatMessage';
+import { pxToRem } from '../../../../utils';
+import { screenReaderContainerStyles } from '../../../../utils/accessibility/Styles/accessibilityStyles';
+import { ChatMessageVariables } from './chatMessageVariables';
+
+const displayActionMenu = (overlayZIndex: ICSSInJSStyle['zIndex']): ICSSInJSStyle => ({
+  zIndex: overlayZIndex!,
+  overflow: 'visible',
+  opacity: 1,
+  width: 'auto',
+});
+
+export const chatMessageStylesComfyRefresh: ComponentSlotStylesPrepared<
+  ChatMessageStylesProps,
+  ChatMessageVariables
+> = {
+  root: ({ props: p, variables: v, theme }): ICSSInJSStyle => {
+    const borderFocusStyles = getBorderFocusStyles({
+      borderRadius: 'inherit',
+      variables: theme.siteVariables,
+    });
+
+    // Fixes the bubble focus border rendering on top of the user avatar
+    delete (borderFocusStyles[':focus-visible'][':before'] as ICSSInJSStyle).zIndex;
+    delete (borderFocusStyles[':focus-visible'][':after'] as ICSSInJSStyle).zIndex;
```

Comment:
Let's modify `getBorderFocusStyles()` to accept `zIndex`:

```suggestion
      variables: theme.siteVariables,
      // Fixes the bubble focus border rendering on top of the user avatar
      zIndex: 'initial',
    });
```

In `packages/fluentui/react-northstar/src/themes/teams/getBorderFocusStyles.ts`

```diff
  focusOuterBorderWidth?: string;
  borderPadding?: string | BorderPadding;
+ zIndex?: string;
};
const defaultColor = 'transparent';
```

```diff
    focusInnerBorderWidth = sv.focusInnerBorderWidth || sv.borderWidth,
    focusOuterBorderWidth = sv.focusOuterBorderWidth || sv.borderWidth,
+   zIndex = sv.zIndexes.foreground,
  } = args;
  const defaultPreudoStyles: React.CSSProperties = {
```

```diff
        borderWidth: focusInnerBorderWidth,
-       zIndex: sv.zIndexes.foreground,
+       zIndex,
        borderColor: focusInnerBorderColor,
```

```diff
        borderWidth: focusOuterBorderWidth,
-       zIndex: sv.zIndexes.foreground,
+       zIndex,
        borderColor: focusOuterBorderColor,
```

Reply:
That's much better, thank you. Fixed.